cmake/dependencies.cmake

```diff
@@ -6,7 +6,16 @@ set_directory_properties(PROPERTIES
     EP_PREFIX ${EP_PREFIX}
     )
 
-# Project dependencies.
+if(CMAKE_GENERATOR MATCHES Xcode)
+  set(XCODE_EXT "/Debug")
```

Comment:
Not really a good name, better use `COMPILER_BUILD_DIR`, or smth like that

Reply:
semantic is "directory extention when cmake generator is xcode"

Reply:
My first association is a file extension, moreover it used like that